core/src/main/java/edu/jhuapl/dorset/Application.java

```diff
@@ -122,35 +124,46 @@ public void addResponseFilter(ResponseFilter filter) {
     /**
      * Add a shutdown listener
      *
-     * @param listener  a listener that runs when shutdown() is called
+     * @param listener a listener that runs when shutdown() is called
      */
     public void addShutdownListener(ShutdownListener listener) {
         shutdownListeners.add(listener);
     }
 
+    public void setUser(User user) {
```

Comment:
This method needs javadoc comment - tell them when should this be used (only for single user applications)